modules/git/hook.go

```diff
@@ -107,9 +109,11 @@ func ListHooks(repoPath string) (_ []*Hook, err error) {
 		return nil, errors.New("hooks path does not exist")
 	}
 
-	hooks := make([]*Hook, len(hookNames))
-	for i, name := range hookNames {
-		hooks[i], err = GetHook(repoPath, name)
+	hookNames := []string{"pre-receive", "update", "post-receive"}
```

Comment:
Regarding maintainability, it's probably a better idea to move this variable right above/below `func GetHookNames()`